.env-example

```diff
@@ -9,4 +9,8 @@ EMAIL_USE_TLS=
 CITESPHERE_ENDPOINT=
 CITESPHERE_CLIENT_ID=
 CITESPHERE_CLIENT_SECRET=
```

Comment:
those three above  should not be needed anymore

Reply:
*`CITESPHERE_ENDPOINT`** is still being used in the **dashboard** for making API calls to Citesphere, specifically in views such as:

- **`get_citesphere_groups(request)`**
- **`get_citesphere_collections(request, group_id)`**

Additionally, **`CITESPHERE_CLIENT_ID`** and **`CITESPHERE_CLIENT_SECRET`** are still required for the **OAuth login flow** in the following views:

- **`citesphere_login(request)`**
- **`citesphere_callback(request)`**

Reply:
no, these should all be configured when adding a repository through admin. there should be no general citesphere instance configured.